components/mpas-ocean/src/mode_init/Makefile

```diff
@@ -84,6 +84,10 @@ mpas_ocn_init_hurricane.o: $(UTILS)
 
 mpas_ocn_init_tidal_boundary.o: $(UTILS)
 
+mpas_ocn_init_dam_break.o: $(UTILS)
```

Comment:
Are these Makefile changes part of the PR?

Reply:
I just pushed a new commit that reverts this Makefile change.